src/containers/JudgingPanel.js

```diff
@@ -202,6 +202,23 @@ export default () => {
     await submitGrade(id, { ...score, removed: true }, { uid: gradeId, email: score.user })
   }
 
+  const onDisqualify = async projectId => {
+    try {
+      await db.runTransaction(async transaction => {
+        const projectDoc = await transaction.get(projectsRef.doc(projectId))
+        if (!projectDoc.exists) {
+          alert('Project does not exist')
+          return
+        }
+        const disqualified = !projectDoc.data().disqualified
+        transaction.update(projectsRef.doc(projectId), { disqualified })
+      })
+    } catch (e) {
+      alert(e)
+    }
+    window.location.reload()
```

Comment:
is there no way to refresh the data without reloading?

Reply:
i was lazy cuz its admin - whats the easiest way to refresh you think?

Reply:
honestly this is fine LOL iTs aDmiN